resources/api/reservation.py

```diff
@@ -206,12 +205,20 @@ def validate(self, data):
                 )
             reservable_after = resource.get_reservable_after()
             if reservable_after and data["begin"] < reservable_after:
-                raise ValidationError(
-                    _(
-                        "The resource is reservable only after %(datetime)s"
-                        % {"datetime": reservable_after}
+                # date falls before the reservable after date. If the number of days
+                # difference (based on date rather than datetime) is equal to the min
+                # days in advance, then should still be permitted
+                is_day_before = (
+                    data["begin"].date() - timezone.now().date()
```

Comment:
One case where I think this doesn't work is when a reservation is made on the last day of they month, e.g. when a reservation is made on 31.5.2023 for 1.6.2023.

Reply:
Maybe we should utilize `datetime.timedelta` here?